Source/Urho3D/Network/HttpRequest.h

```diff
@@ -45,6 +45,10 @@ enum HttpRequestState
 /// An HTTP connection with response data stream.
 class URHO3D_API HttpRequest : public RefCounted, public Deserializer, public Thread
 {
+private:

```

Comment:
There is already a `private` block at the end of the class definition and all variable names must start with lower case letter end with and underscore. So in this case it would be `sslInitialized_` + you could also initialize it in the header file with the brackets `sslInitialized_{false}`

Reply:
That's a static member variable and must be initialized in the source file. It's static because `mg_init_library` should only be called once, no matter how many requests you make.

I can change the name of the variable.